src/orion/core/worker/experiment.py

```diff
@@ -153,7 +152,10 @@ def __init__(
         working_dir: str | None = None,
         metadata: dict | None = None,
         refers: dict | None = None,
+        storage=None,
```

Comment:
```suggestion
        storage: BaseStorageProtocol | None = None,
```
That's a bit weird actually, followed by the assert. Making it a positional argument would broken many parts of the code I guess?

Reply:
yeah that is the main problem